packages/foam-vscode/static/graphs/default/graph.js

```diff
@@ -212,8 +225,11 @@ function augmentGraphInfo(data) {
   data.links.forEach(link => {
     const a = data.nodes[link.source];
     const b = data.nodes[link.target];
-    a.neighbors.push(b.id);
-    b.neighbors.push(a.id);
+    // Workaround: nodes of type placeholder have mixed up id and uri
```

Comment:
what is happening here? not sure I understand

Reply:
It's an ugly workaround to a bug in dataviz.ts that is setting id to a dictonary and not a string for placeholder nodoes - sorry :) At the time I hadn't dug deep enough into the code to find the underlying bug. But I think I can now see how to fix this in datavviz.ts  instead - let me work on a better fix and update this PR!

Reply:
Reversed this change and fixed in dataviz.ts now.